api/api/accounts/permissions.py

```diff
@@ -5,4 +5,6 @@ class IsOwnerOrAdmin(permissions.BasePermission):
     """Custom permission to allow only Owners of profile and Admin Users"""
 
     def has_object_permission(self, request, view, obj):
+        if request.method in permissions.SAFE_METHODS:
```

Comment:
This is something different then what the permission name suggests. Why do we need this?

Reply:
> Why do we need this?

I added this so that any user can retrieve any other user's profile. 

> This is something different then what the permission name suggests.

Hmm. I'll add another permission.

Reply:
> I added this so that any user can retrieve any other user's profile.

Do we have a usecase for that yet?

Reply:
I assumed there to be 🤔 . @Aksh-Bansal-dev ?

Reply:
I don't think we need this right now but maybe, in the future, we can allow club officials to view members' profiles.